fc-community-contracts/src/FactChainNFT.sol

```diff
@@ -5,29 +5,45 @@ import {ERC721URIStorage} from "openzeppelin-contracts/contracts/token/ERC721/ex
 import {ERC721} from "openzeppelin-contracts/contracts/token/ERC721/ERC721.sol";
 import {Ownable} from "./utils/Ownable.sol";
 
+contract FactChainSFT {
```

Comment:
maybe retry to import `import {FactChainSFT} from ...` but otherwise this is fine too 👍